code/game/objects/items/storage/boxes.dm

```diff
@@ -119,19 +119,24 @@
 	desc = "A box with the bare essentials of ensuring the survival of you and others."
 	icon_state = "internals"
 	illustration = "emergencytank"
-	var/mask_type = /obj/item/clothing/mask/breath
-	var/internal_type = /obj/item/tank/internals/emergency_oxygen
+	var/mask_type = /obj/item/clothing/mask/gas
+	var/internal_type = null
 	var/medipen_type = /obj/item/reagent_containers/hypospray/medipen
+	var/filter_type = /obj/item/gas_filter
```

Comment:
This line does not meet our contribution standards, specifically for [dmdoc compliance](https://github.com/tgstation/tgstation/blob/master/.github/AUTODOC_GUIDE.md), and should be refactored.